### PR TITLE
Run3-alca247Z Add a new defintion of truncated DetID and a new scaling script for HCAL IsoTrack  calibration

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Add a new defintion of truncated DetID and a new scaling script for HCAL IsoTrack  calibration

#### PR validation:

Used the scripts for getting calibration HCAL IsoTrack constants 

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Calibration/HcalCalibAlgos/macros/CalibCorr.C [hunk 1]
```diff
@@ -144,6 +148,12 @@ unsigned int truncateId(unsigned int detId, int truncateFlag, bool debug = false
     //Ignore depth index for depth > 1 in HB and HE
     if (depth > 1)
       depth = 2;
+  } else if (truncate0 == 6) {
+    //Ignore depth index for depth > 1 in HB and HE
```
Comment (perrotta) [inline Calibration/HcalCalibAlgos/macros/CalibCorr.C:152]: The comment is the same as the one at [L148](https://github.com/cms-sw/cmssw/pull/45044/files#diff-d54af1ffe7c3b1c015cba727cd7149a6a3ca54a355ffdeb404196427970244beR148), but the range of `depth`'s is different: please fix the comment

File: Calibration/HcalCalibAlgos/macros/CalibPlotProperties.C [hunk 2]
```diff
@@ -74,8 +74,9 @@
 //                               all depths as depth 1 (2), all depths in HE
 //                               with values > 1 as depth 2 (3), all depths in
 //                               HB with values > 1 as depth 2 (4), all depths
-//                               in HB and HE with values > 1 as depth 2 (5)
-//                               (Default 0)
+//                               in HB and HE with values > 1 as depth 2 (5),
+//                               for depth = 1 and 2, depth = 1, else depth
+//                                = 2. (Default 0)
```
Comment (perrotta) [inline Calibration/HcalCalibAlgos/macros/CalibPlotProperties.C:79]:
```suggestion
//                                = 2 (6). (Default 0)
```

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
please test
(errors are clearly unrelated, but let see if they reset if we relaunch the tests)

**Comment by perrotta** [comment]
+alca

**Pull request merged**